UnitTests/BraintreePayPalTests/BTPayPalRequest_Tests.swift

```diff
@@ -46,8 +45,12 @@ class BTPayPalRequest_Tests: XCTestCase {
         request.riskCorrelationID = "123-correlation-id"
         request.merchantAccountID = "merchant-account-id"
         request.isShippingAddressEditable = true
-
-        request.lineItems = [BTPayPalLineItem(quantity: "1", unitAmount: "1", name: "item", kind: .credit)]
+        
+        let lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "10", name: "item-name", kind: BTPayPalLineItemKind.debit);
+        lineItem.imageURL = URL(string: "http://example/image.jpg");
+        lineItem.upcCode = "upc-code";
+        lineItem.upcType = BTPayPalLineItemUPCType.UPC_A;
```

Comment:
nit since some of this syntax can be cleaned up to be more "swift-y":
```suggestion
        let lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "10", name: "item-name", kind: .debit)
        lineItem.imageURL = URL(string: "http://example/image.jpg")
        lineItem.upcCode = "upc-code"
        lineItem.upcType = .UPC_A
```